python/paddle/fluid/tests/unittests/test_unsqueeze_op.py

```diff
@@ -0,0 +1,229 @@
+        self.check_output()
+
+    def test_check_grad(self):
+        self.check_grad(["X"], "Out")
+
+
+# Correct: Inplace. There is duplicated axis.
+class TestUnsqueezeOpInplace3(OpTest):
+    def setUp(self):
+        ori_shape = (3, 2, 5)
+        axes = (0, 3, 3)
+        new_shape = (1, 3, 2, 1, 1, 5)
+
+        self.op_type = "unsqueeze"
+        self.inputs = {"X": np.random.random(ori_shape).astype("float32")}
+        self.attrs = {"axes": axes, "inplace": True}
+        self.outputs = {"Out": self.inputs["X"].reshape(new_shape)}
+
+    def test_check_output(self):
+        self.check_output()
+
+    def test_check_grad(self):
+        self.check_grad(["X"], "Out")
+
+
+'''
+# Error: Output dimension is error.
+class TestUnsqueezeOp4(OpTest):
+    def setUp(self):
+        ori_shape = (3, 5)
+        axes = (0, 3)
+        new_shape = (1, 3, 1, 1, 5)
+
+        self.op_type = "unsqueeze"
+        self.inputs = {"X": np.random.random(ori_shape).astype("float32")}
+        self.attrs = {"axes": axes, "inplace": False}
+        self.outputs = {"Out": self.inputs["X"].reshape(new_shape)}
+
+    def test_check_output(self):
+        self.check_output()
+
+    def test_check_grad(self):
+        self.check_grad(["X"], "Out")
+
+# Error: Input axis is large than output range.
+class TestUnsqueezeOp5(OpTest):
+    def setUp(self):
+        ori_shape = (3, 5)
+        axes = (0, 4)
+        new_shape = (1, 3, 5, 1)
+
+        self.op_type = "unsqueeze"
+        self.inputs = {"X": np.random.random(ori_shape).astype("float32")}
+        self.attrs = {"axes": axes, "inplace": False}
+        self.outputs = {"Out": self.inputs["X"].reshape(new_shape)}
+
+    def test_check_output(self):
+        self.check_output()
+
+        def test_check_grad(self):
+            self.check_grad(["X"], "Out")
+
+# Error: Input axes is large than Eigen limit.
+class TestUnsqueezeOp6(OpTest):
+    def setUp(self):
+        ori_shape = (3, 5)
+        axes = (0, 2, 10)
+        new_shape = (1, 3, 1, 5, 1)
+
+        self.op_type = "unsqueeze"
+        self.inputs = {"X": np.random.random(ori_shape).astype("float32")}
+        self.attrs = {"axes": axes, "inplace": False}
+        self.outputs = {"Out": self.inputs["X"].reshape(new_shape)}
+
+    def test_check_output(self):
+        self.check_output()
+
+    def test_check_grad(self):
+        self.check_grad(["X"], "Out")
+
+# Error: Input axes size is large than Eigen limit.
+class TestUnsqueezeOp7(OpTest):
+    def setUp(self):
+        ori_shape = (3, 5)
+        axes = (0, 2, 2, 2, 2, 2)
+        new_shape = (1, 3, 1, 1, 5, 1)
+
+        self.op_type = "unsqueeze"
+        self.inputs = {"X": np.random.random(ori_shape).astype("float32")}
+        self.attrs = {"axes": axes, "inplace": False}
+        self.outputs = {"Out": self.inputs["X"].reshape(new_shape)}
+
+    def test_check_output(self):
+        self.check_output()
+
+    def test_check_grad(self):
+        self.check_grad(["X"], "Out")
+'''
```

Comment:
If the annotated code is useless, you should remove them.

Reply:
ok, I will remove them